docs/api-overview.md

```
@@ -2414,6 +2414,7 @@ _Appears in:_
 | Field | Description | Default | Validation |
 | --- | --- | --- | --- |
 | `monitoring` _[ManagementSpec](#managementspec)_ | Monitoring controls the monitoring module operator lifecycle. |  |  |
+| `kserve` _[ManagementSpec](#managementspec)_ | Kserve controls the kserve module operator lifecycle. |  |  |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)